ansible/inventory.yml

```diff
@@ -51,9 +51,9 @@ hosts:
         ibmi73-ppc64_be-1: {ip: 65.183.160.62, user: nodejs}
 
     - joyent:
```

Comment:
does this have to stay "joyent" ?

Reply:
No, it should change, but I ran out of time before going on leave for Christmas and the New Year:

> We'll need to rename the release and infra machines too but I won't have time to do that this year.

I want to keep the names in the inventory and Jenkins in sync. Renaming the hosts in Jenkins will result in a new secret which means updating the secrets repository and reansibling the machines.